### PR TITLE
update readme for configuration file analysis

### LINKED ISSUE #3: Update readme
- [ ] Acceptance: Read me file with the following:

1. Some explanatory content from the blog
2. Relevant notebooks and two lines explaining what each of them is for

### PR DESCRIPTION
**Pull request by SankBad**
This PR will update the README of the repository. Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,39 @@
-project-template
-==============================
+# Configuration file analysis
 
-template for the team to use
+## Overview
+
+Today's software system became more flexible and feature-rich. For example, the configuration file for MySQL has more than 200 configuration entries with different subentries.  As a result, configuring these systems is a complicated task and frequently causes configuration errors. Currently, in most cases, misconfigurations are detected by manually specified rules. However, this process is tedious and not scalable. In this project, we propose data-driven methods to detect misconfigurations by discovering frequently occurring patterns in configuration files.
```
Comment (Shreyanand) [inline README.md]: Software systems have become more flexible and feature-rich. ...

**Review by Shreyanand — CHANGES_REQUESTED** [review]
LGTM, minor change

**Comment by SankBad** [comment]
Thanks for the feedback @Shreyanand :smile: 
I fixed a typo.

**Comment by SankBad** [comment]
Also, squashed commits.

**Review by Shreyanand — APPROVED** [review]
:rocket:

**Pull request merged**